licenses/library.go

```diff
@@ -82,6 +82,13 @@ func Libraries(ctx context.Context, classifier Classifier, importPaths ...string
 			// No license requirements for the Go standard library.
 			return false
 		}
+		for _, i := range ignoredPaths {
+			if strings.HasPrefix(p.PkgPath, i) {
+				// Marked to be ignored.
```

Comment:
nit: Log an info here?

Reply:
Adding a log here introduces a lot of `ignored` log lines for every sub-packages under the ignored paths.
I feel it's fine to be silently ignored since it is explicitly specified to be ignored.
What do you think?

Reply:
On second thought, yes that makes sense to me.